src/gui/editors/SongEditor.cpp

```diff
@@ -988,19 +1006,29 @@ SongEditorWindow::SongEditorWindow(Song* song) :
 
 	DropToolBar *zoomToolBar = addDropToolBarToTop(tr("Zoom controls"));
 
-	auto zoom_lbl = new QLabel(m_toolBar);
-	zoom_lbl->setPixmap( embed::getIconPixmap( "zoom" ) );
+	auto zoomx_lbl = new QLabel(m_toolBar);
+	zoomx_lbl->setPixmap(embed::getIconPixmap("zoom_x"));
+	auto zoomy_lbl = new QLabel(m_toolBar);
```

Comment:
`zoomx_lbl` and `zoomy_lbl` don't follow our naming conventions. Maybe `zoomXLabel` and `zoomYLabel` could be used.